src/main/java/net/sf/jsqlparser/util/deparser/DeleteDeParser.java

```diff
@@ -107,6 +104,13 @@ public void deParse(Delete delete) {
         }
 
     }
+    
+    protected void acceptWhere(Delete delete) {
```

Comment:
ok, but please rename into something like `deparsWhereClause` to stick with the established naming convention.
`accept` implies the visitor pattern which is not applicable here.